suitcase/tiff/__init__.py

```diff
@@ -101,7 +112,15 @@ def export(gen, filepath, **kwargs):
                     stream_name = stream_names[event_page['descriptor']]
                     for field in event_page['data']:
                         for img in event_page['data'][field]:
-                            files[event_page['descriptor']].save(img, *kwargs)
+                            # check that the data is 2D, if not raise exception
+                            if len(numpy.asarray(img).shape) == 2:
```

Comment:
:+1: for using `asarray` here. I suggest using `.ndim` instead of `len(...).shape`.

Reply:
done